.github/workflows/env/action.yml

```
@@ -28,10 +28,28 @@ runs:
         sudo apt-get update -y
 
         sudo apt-get install -y curl unzip gcc-aarch64-linux-gnu \
-          libc6-arm64-cross qemu-user-binfmt libc6:arm64
+          libc6-arm64-cross qemu-user-binfmt libc6:arm64 \
+          musl-dev:amd64 musl-dev:arm64 musl-tools binutils-aarch64-linux-gnu
     - name: Set up Go
       uses: actions/setup-go@v5
       with:
         go-version-file: go.mod
         cache-dependency-path: go.sum
       id: go
+    - name: Install Rust
+      uses: dtolnay/rust-toolchain@stable
+      with:
+        targets: x86_64-unknown-linux-musl,aarch64-unknown-linux-musl
+    - name: Install protoc
+      shell: bash
+      env:
+        PB_URL: "https://github.com/protocolbuffers/protobuf/releases/download/v24.4/"
+        PB_FILE: "protoc-24.4-linux-x86_64.zip"
+        INSTALL_DIR: "/usr/local"
+      run: |
+        wget -q "$PB_URL/$PB_FILE"
```

`-q` also suppresses the error message. `-nv` is like `-q` but with error messages printed out.
```suggestion
        wget -nv "$PB_URL/$PB_FILE"
```